test/helpers/contract-helpers.ts

```diff
@@ -59,35 +93,55 @@ export const DataGenerator = {
       }
       return result;
     },
-    byId: (id: any) => DB.clusters[id].operatorIds
-  }
+    byId: (id: any) => DB.clusters[id].operatorIds,
+  },
+};
+
+export const getClusterForValidator = (
+  validatorCount: number,
+  networkFeeIndex: number,
+  index: number,
+  balance: number,
+  active: boolean,
+) => {
+  return { validatorCount, networkFeeIndex, index, balance, active };
 };
 
 export const initializeContract = async () => {
   CONFIG = {
-    initialVersion: "v1.0.2",
+    initialVersion: 'v1.1.0',
     operatorMaxFeeIncrease: 1000,
     declareOperatorFeePeriod: 3600, // HOUR
     executeOperatorFeePeriod: 86400, // DAY
     minimalOperatorFee: 100000000,
     minimalBlocksBeforeLiquidation: 100800,
     minimumLiquidationCollateral: 200000000,
     validatorsPerOperatorLimit: 500,
-    maximumOperatorFee: 76528650000000
+    maximumOperatorFee: 76528650000000,
   };
 
   DB = {
     owners: [],
     validators: [],
     operators: [],
+    registeredValidators: [],
+    registeredOperators: [],
     clusters: [],
     ssvNetwork: {},
     ssvViews: {},
     ssvToken: {},
     ssvOperatorsMod: {},
     ssvClustersMod: {},
     ssvDAOMod: {},
-    ssvViewsMod: {}
+    ssvViewsMod: {},
+    ownerNonce: 0,
+    initialClusterState: {
```

Comment:
@Mohsen-T same here, do we have a helper function for initiating a cluster?

Reply:
I've reviewed the implementation of the `bulkRegisterValidators` function and can suggest an enhancement regarding the handling of the 5th parameter. 
Instead of using `initialClusterState`, it would be more efficient and cleaner to utilize the `getClusterForValidator` function from contract-helpers.ts.

You can find the [getClusterForValidator](https://github.com/bloxapp/ssv-network/blob/v1.1.0/test/helpers/contract-helpers.ts#L93) function on the v1.1.0 branch.
It can be seamlessly integrated into the current setup as follows:

`helpers.getClusterForValidator(0, 0, 0, 0, true)`